src/requirements.in

```diff
@@ -1,11 +1,12 @@
---extra-index-url https://pypi.org/simple/
-
 Sphinx
 
 # At least 4.2.0, as it fixes a bug in combination with esbonio live preview:
 # https://github.com/useblocks/sphinx-needs/issues/1350
 sphinx-needs>=4.2.0
 
+# "sphinx-collections" pypi release is ancient
```

Comment:
Btw, I brought the package up-to-date last week
https://github.com/useblocks/sphinx-collections/pulls?q=is%3Apr+is%3Aclosed
Just need to release. There are no functional changes. But the new package classifiers should be deployed actually.

Reply:
oh it is on pypi! We can remove the workaround!